drivers/segger/note_sysview.c

```diff
@@ -155,11 +159,24 @@ static bool sysview_isenabled(void)
     }
 #endif
 
+#ifdef CONFIG_SCHED_INSTRUMENTATION_SYSCALL
+  /* Use the Syscall "0" to identify whether the syscall is enabled,
```

Comment:
but how to handle if nobody call syscall#0?

Reply:
The 0th syscall is a special symbol (_exit()), most calls focus only on exit()

https://github.com/apache/incubator-nuttx/blob/master/include/sys/syscall_lookup.h#L27